web_timeline/static/src/js/timeline_view.js

```diff
@@ -111,12 +111,18 @@ odoo.define('web_timeline.TimelineView', function (require) {
             if (!isNullOrUndef(attrs.quick_create_instance)) {
                 self.quick_create_instance = 'instance.' + attrs.quick_create_instance;
             }
+            if (isNullOrUndef(attrs.stack) || !_.str.toBoolElse(attrs.stack, "true")) {
+                this.stack = false;
```

Comment:
By default it should stack IMO.

Reply:
The toBoolElse function is used the wrong way here (by reading the definition). 
Also you can remove the else by doing it like this:

```
            this.stack = true;
            if (!isNullOrUndef(attrs.stack) && !_.str.toBoolElse(attrs.stack, true)) {
                this.stack = false;
            }
```